onnx2torch/node_converters/resize.py

```diff
@@ -0,0 +1,188 @@
+__all__ = ['OnnxResize']
+
+import warnings
+
+import torch
+from torch import nn
+from typing import Optional
+
+from onnx2torch.common import OperationConverterResult
+from onnx2torch.common import onnx_mapping_from_node
+from onnx2torch.node_converters.registry import add_converter
+from onnx2torch.onnx_graph import OnnxGraph
+from onnx2torch.onnx_node import OnnxNode
+
+LINEAR_MODES = {1: 'linear', 2: 'bilinear', 3: 'trilinear'}
+
+
+def _get_torch_align_corners(mode: str, coordinate_transformation_mode: str) -> Optional[bool]:
+    align_corners = coordinate_transformation_mode == 'align_corners'
+    if mode == "nearest":
+        return None
+    return align_corners
+
+
+def _dimension_mode(mode: str, dim_size: int) -> str:
+    if dim_size not in [1, 2, 3]:
+        raise RuntimeError('Input tensor for resize has to have from 1 to 3 dimensions.')
+
+    if mode == 'nearest':
+        return mode
+
+    elif 'cubic' in mode:
+        if dim_size == 2:
+            return 'bicubic'
+        else:
+            raise NotImplementedError(f"{dim_size}D input is not implemented for {mode} mode.")
+
+    elif 'linear' in mode:
+        return LINEAR_MODES[dim_size]
+
+    else:
+        raise RuntimeError(f'Got unexpected mode for interpolation ({mode}).')
+
+
+class OnnxResize(nn.Module):
+
+    def __init__(
+            self,
+            coordinate_transformation_mode: str = 'asymmetric',
+            mode: str = 'nearest',
+    ):
+        super().__init__()
+        self.mode = mode
+        self.coordinate_transformation_mode = coordinate_transformation_mode
+        self.align_corners = _get_torch_align_corners(self.mode, self.coordinate_transformation_mode)
+
+    def forward(
+            self,
+            input_tensor: torch.Tensor,
+            roi: torch.Tensor = torch.tensor([]),
+            scales: torch.Tensor = torch.tensor([]),
+            sizes: torch.Tensor = torch.tensor([]),
```

Comment:
mutable objects in default parameters is always bad idea

Reply:
Done.